pkg/scanners/trivy/trivy.go

```diff
@@ -0,0 +1,399 @@
+package main
```

Comment:
as we discussed before, let's add unit tests

we can add e2e once we have the collector logic